### PR TITLE
Looking for feedback! :) 

### PR DESCRIPTION
**Pull request by butlersrepos**
Updated the README with setup instructions. Added shebang to bin entry file. Formatted src. Upped minor version. Changed package json target bin. Upgraded the wd version

### CODE REVIEW

**Comment by Jonahss** [comment]
super! I'll review right now.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,122 @@
...
+1. `npm install -g appium-repl`
+2. create your configuration `.appium-repl.json`
+3. run the `appium` server
+  * find appium [here](https://github.com/appium/appium)
+  * ensure that you have the appropriate XCode/Android SDKs installed
+  * run it, _consult appium's documentation if any issues occur_
+4. run `appium-repl` 
+5. choose your configuration option (`.appium-repl.json` can have any number of defined configurations)
+6. access `driver` directly from the REPL. refer to [wd docs](https://github.com/admc/wd/blob/master/doc/api.md) for relevant methods
+7. profit?
```
Comment (Jonahss) [inline README.md]: :trollface:

**Comment by Jonahss** [comment]
Looks good :) Thanks for the contribution. Any ideas of what could be added beyond this?

**Comment by Jonahss** [comment]
published as v0.1.0

**Comment by butlersrepos** [comment]
I'm not sure. This is a new area for me, currently I'm using appium to test a cordova app on iOS and Android and right now this tool helps get past the boilerplate of connecting and launching nicely.

Any improvements that allow you to configure away the minutiae (put em under version control too) and be able to immediately get into poking at the app with `wd` are great. Writing functional tests with wd is pretty abysmal since its api is poorly documented, sometimes unimplemented, and obtuse even when it does what it says.

I plan on continuing to use this in the immediate future so I'll definitely keep it in mind and hopefully have more to say as I explore it.

**Comment by Jonahss** [comment]
Try [webdriver.io](http://webdriver.io/) as an alternative to wd.js

**Pull request merged**